assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.stories.js

```diff
@@ -59,6 +59,10 @@ Default.storyName = 'Default';
 Default.scenario = {};
 Default.args = {
 	setupRegistry: ( registry ) => {
+		publications[ 0 ].products.push( {
```

Comment:
This is mutating an object which is in the Redux state, which should be avoided.

I'm not entirely clear what this particular case is trying to achieve, but it should be rewritten or removed.

Reply:
This will be no longer required because of above change in the selector.

Reply:
Sorry for confusion, this will be required and I have fixed it as per the suggestion.